Formula/yadm.rb

```diff
@@ -1,21 +1,44 @@
 class Yadm < Formula
   desc "Yet Another Dotfiles Manager"
-  homepage "https://thelocehiliosan.github.io/yadm/"
-  url "https://github.com/TheLocehiliosan/yadm/archive/1.12.0.tar.gz"
-  sha256 "c3d612d01e2027d5f457e0f7d120bc67251b716c373d99fe70638bd86edf107f"
+  homepage "https://yadm.io/"
+  url "https://github.com/TheLocehiliosan/yadm/archive/2.0.0.tar.gz"
+  sha256 "6359debdd9a6154709d084f478c000e572b3d8d50c2abe6525534899a5c2eb16"
 
   bottle :unneeded
 
   def install
+    post_install_message = any_version_installed?
     bin.install "yadm"
     man1.install "yadm.1"
     bash_completion.install "completion/yadm.bash_completion"
     zsh_completion.install  "completion/yadm.zsh_completion" => "_yadm"
+    doc.install "CHANGES"
+    doc.install "CONTRIBUTORS"
+    doc.install "LICENSE"
+    doc.install "contrib"
+    if post_install_message
```

Comment:
This should be documented in the software itself, not in our formula. We only use messages/caveat for things that are specific to Homebrew.

Reply:
I can remove this. I put the message in because upgrading from 1.x.x has some major interface changes that are not backwards compatible.

Reply:
Unless those only occur in homebrew they shouldn't be shown by homebrew.